src/components/BlogFeed.astro

```diff
@@ -67,6 +70,7 @@ type ArticleResult = {
             <img src={image.src} width={image.width} alt={media.data.alt} />
           </figure>
           <h4>{post.webTitle}</h4>
+          <h5><em>{post.fields.byline}</em></h5>
           <h5>{pubDate}</h5>
```

Comment:
I don't have the answer here, but are `h5`s (or headings in general) the best element to use for author and publication date info?

Reply:
This is a very good point, and I think we should consider alternatives. But probably one for a separate PR!

Reply:
Probably something like `<time>` and `<address rel="author">`.